tests/unittests/hrm2_tests.py

```
@@ -0,0 +1,1336 @@
+        self.assertEqual(person["person_id"], 0)
+        self.assertIsNone(person["bbox"])  # single-person mode
+
+        # Verify camera_translation was computed (not None)
+        self.assertIn("camera_translation", person)
+        self.assertIsNotNone(person["camera_translation"])
+
+
+class HRM2DatasetTests(HRM2TestBase):
+    """Tests for HRM2Model dataset application."""
+
+    def _make_dataset(self):
```

Ruff (0.14.5)

Missing return type annotation for private function `_make_dataset`

(ANN202)